__tests__/db.test.js

```diff
@@ -51,3 +51,31 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200: responds with an article object by ID", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
```

Comment:
You can be more specific than just testing the properties exist as you know what exactly article you are testing for